http-api/src/main/java/net/runelite/http/api/item/ItemStats.java

```diff
@@ -30,6 +30,16 @@
 @Value
 public class ItemStats
 {
+	// Unarmed attack speed is 4
+	public static final ItemStats UNARMED = new ItemStats(false, true, 0, 0,
```

Comment:
This shouldn't be stored here since it's not used in this class. It should probably continue to remain in the overlay class.